partiql-lang/src/main/pig/partiql.ion

```diff
@@ -434,10 +434,18 @@ may then be further optimized by selecting better implementations of each operat
         // A data definition operation.
         (sum ddl_op
             // `CREATE TABLE <symbol> ( <table_def> )?`
-            (create_table table_name::symbol def::(? table_def))
+            // We support variable length namespace
+            // For example, in SQL, we can have
+            // 1) CREATE TABLE <catalog name>.<schema name>.<table name> ...
+            // 2) CREATE TABLE <schema name>.<table name> ...
+            // 3) CREATE TABLE <table name>
+            // however, CREATE TABLe statement in the above assumes the existences of all path steps other than the table
+            // For example, in case 1, the catalog name needs to be a existing catalog and the schema name needs to be a existing schema.
+            // which means all the "steps" in create table statement, except for the last one, should be identifer (i.e., case sensitivity matters)
+            (create_table table_name_prefix::table_name_prefix table_name::symbol def::(? table_def))
 
             // `DROP TABLE <identifier>`
-            (drop_table table_name::identifier)
+            (drop_table table_name_prefix::table_name_prefix table_name::identifier)
```

Comment:
`table_name_prefix`+`identifier` here should become one `qualified_identifier` eventually -- in that "rationalization" PR.

Reply:
I opted to make the prefix optional, let me know your opinion on this.